Xamarin.Essentials/Platform/Platform.android.cs

```diff
@@ -180,8 +180,13 @@ internal static AndroidUri GetShareableFileUri(string filename)
             false;
 #endif
 
+        static int? sdkInt;
+
+        internal static int SdkInt
+            => sdkInt ?? (sdkInt = (int)Build.VERSION.SdkInt).Value;
```

Comment:
```suggestion
            => sdkInt ??= (int)Build.VERSION.SdkInt;
```

Gotta use the new things!